caffe2/python/onnx/frontend.py

```diff
@@ -504,14 +539,25 @@ def _annotate_consumed(cls, graph_def):
                     consumes,
                 )])
 
+    @classmethod
+    def _filter_fake_init(cls, init_net, value_info):
+        if init_net:
+            fake_inits = [op for op in init_net.op
+                          if len(op.output) == 1 and op.output[0] in value_info and
+                          re.match('GivenTensor.*Fill|ConstantFill', op.type)]
+            for fake_init in fake_inits:
```

Comment:
You can do 
```
init_net.op = filter(lambda x: x not in fake_inits, init_net.op)
````
to replace everything from here.

Reply:
The problem here is... you cannot assign anything to init_net.op. The only way I have figured out to change op, is the remove and del.